### ISSUE TITLE
[$75] IOS Hybrid - Copilot - Not so fast modal becomes translucent on security page

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#53833]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.73-6
**Reproducible in staging?:** Yes
**Reproducible in production?:** No
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** No, reproducible on hybrid only
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://github.com/Expensify/App/pull/52103
**Email or phone of affected tester (no customers):** nathanmulugetatesting+2373@gmail.com
**Issue reported by:** Applause Internal Team

## Action Performed:
Precondition: Sign in with a copilot account

1. Open Expensify hybrid app
2. Navigate to settings > Security
3. Tap on Two factor authentication 
4. Once the Not so fast modal shows up quickly close it and tap on Close account
5. Do step 3 and 4 quickly until issue is seen (Mostly seen after 4-5 tries)

## Expected Result:
Not so fast modal background doesn't change

## Actual Result:
Not so fast modal background becomes translucent

## Workaround:
Unknown

## Platforms:
- [ ] Android: Standalone
- [ ] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [x] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/6ba24dd4-607b-44ee-8bee-7d5ba59941fa

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021866868576432748214</li>
        <li>Upwork Job ID: 1866868576432748214</li>
        <li>Last Price Increase: 2025-01-01</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @allroundexperts</details>

**Comment by twisterdotcom** [comment]
>Once the Not so fast modal shows up quickly close it and tap on Close account
>Do step 3 and 4 quickly until issue is seen (Mostly seen after 4-5 tries)

I don't think we should be blocking a deploy on incredibly unlikely user behaviour like this. I am fine with understanding the issue and fixing it if we truly care (I don't personally think we should). @grgia intrigued for your take too.

**Comment by luacmartins** [comment]
I agree. Removing the blocker label

**Comment by grgia** [comment]
Yep thank you!

**Comment by twisterdotcom** [comment]
Can we close this?

**Comment by grgia** [comment]
@twisterdotcom the bug still exists, I'll just lower the bounty to reflect the priority and add help wanted label.

**Comment by twisterdotcom** [comment]
Making weekly.

**Comment by twisterdotcom** [comment]
Making weekly

**Comment by QichenZhu** [comment]
Similar to https://github.com/Expensify/App/issues/51869.

I'm guessing regular contributors can't work on this since it's a hybrid app issue, right?

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by twisterdotcom** [comment]
Going to close.
